modules/meta/arrow-meta/src/main/java/arrow/common/utils/AbstractProcessor.kt

```diff
@@ -44,25 +44,35 @@ abstract class AbstractProcessor : KotlinAbstractProcessor(), ProcessorUtils, Ko
     }
 
   private fun processElementDoc(e: Element) {
-    val doc = elementUtils.getDocComment(e)
-    val kDocLocation = e.kDocLocation()
-    if (doc != null && doc.trim { it <= ' ' }.isNotEmpty()) {
-      @Suppress("SwallowedException")
-      try {
-        val path = kDocLocation.toPath()
-        @Suppress("SwallowedException")
-        try { Files.createDirectories(path.parent) } catch (e: IOException) {
-        }
-        @Suppress("SwallowedException")
-        try { Files.delete(path) } catch (e: IOException) {
-        }
+    try {
+      val doc = elementUtils.getDocComment(e)
+      val kDocLocation = e.kDocLocation()
+      if (doc != null && doc.trim { it <= ' ' }.isNotEmpty()) {
         @Suppress("SwallowedException")
-        try { Files.createFile(path) } catch (e: IOException) {
+        try {
+          val path = kDocLocation.toPath()
+          @Suppress("SwallowedException")
+          try {
+            Files.createDirectories(path.parent)
+          } catch (e: IOException) {
+          }
+          @Suppress("SwallowedException")
+          try {
+            Files.delete(path)
+          } catch (e: IOException) {
+          }
+          @Suppress("SwallowedException")
+          try {
+            Files.createFile(path)
+          } catch (e: IOException) {
+          }
+          kDocLocation.writeText(doc)
+        } catch (x: IOException) {
+          logW("Failed to generate kdoc file location: $kDocLocation", e)
         }
-        kDocLocation.writeText(doc)
-      } catch (x: IOException) {
-        logW("Failed to generate kdoc file location: $kDocLocation", e)
       }
+    } catch (e: Exception) {
+      logE(e.localizedMessage)
```

Comment:
....can't we use Arrow for this?

Reply:
No Arrow on the processors because the processors are used to build Arrow and we never figured out the `shadow` biz.